server/webapp/WEB-INF/rails.new/spec/controllers/home_controller_spec.rb

```diff
@@ -0,0 +1,36 @@
+##########################GO-LICENSE-START################################
+# Copyright 2014 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################GO-LICENSE-END##################################
+
+require File.expand_path(File.dirname(__FILE__) + '/../spec_helper')
+
+describe HomeController do
+  describe "index" do
+
+    it "should resolve" do
+      expect({:get => "/home"}).to route_to(:controller => "home", :action => "index")
```

Comment:
Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.